packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyModelSchemaTest.kt

```diff
@@ -0,0 +1,518 @@
+                                        "association" to mapOf(
+                                                "associationType" to 2,
+                                                "targetName" to "postID"
+                                        ),
+                                        "authRules" to emptyList<Any>()
+                                ),
+                                "content" to mapOf(
+                                        "name" to "content",
+                                        "type" to mapOf(
+                                                "fieldType" to 0,
+                                                "ofModelName" to ""),
+                                        "isRequired" to true,
+                                        "isArray" to false,
+                                        "association" to mapOf<String, Any>(),
+                                        "authRules" to emptyList<Any>()
+                                )
+                        )
+                )
+
+        var modelSchema = FlutterModelSchema(inputMap)
+
+
+        // Build ModelSchema
+        var native = ModelSchema.builder()
+        native.name("Comment")
+        native.pluralName("Comments")
+        native.fields(mapOf(
+                "id" to
+                        ModelField.builder()
+                                .name("id")
+                                .javaClassForValue(String::class.java)
+                                .targetType("String")
+                                .isRequired(true)
+                                .isArray(false)
+                                .build(),
+                "post" to
+                        ModelField.builder()
+                                .name("post")
+                                .javaClassForValue(Model::class.java)
+                                .targetType("Post")
+                                .isRequired(false)
+                                .isArray(false)
+                                .isModel(true)
+                                .build(),
+                "content" to
+                        ModelField.builder()
+                                .name("content")
+                                .javaClassForValue(String::class.java)
+                                .targetType("String")
+                                .isRequired(true)
+                                .isArray(false)
+                                .build()
+        ))
+        native.associations(mapOf(
+                "post" to
+                        ModelAssociation.builder()
+                                .name("BelongsTo")
+                                .targetName("postID")
+                                .build()
+        ))
+
+        // Verify result
+        assertEquals(
+                modelSchema.convertToNativeModelSchema(),
+                native.build()
+        )
+
+    }
+
+
+
+
+
```

Comment:
I would go through this PR and remove any extra lines. You should have at most one empty line around the code.  And `}` should almost never be preceded by an empty line.

Reply:
Sounds good thanks